src/Options/Applicative/Help/Core.hs

```diff
@@ -115,8 +121,8 @@ missingDesc = briefDesc' False
 -- | Generate a brief help text for a parser, allowing the specification
 --   of if optional arguments are show.
 briefDesc' :: Bool -> ParserPrefs -> Parser a -> Chunk Doc
-briefDesc' showOptional pprefs =
-  wrapOver NoDefault MaybeRequired
+briefDesc' showOptional pprefs = id
+    . wrapOver NoDefault MaybeRequired
```

Comment:
why compose the identity function?

Reply:
Stylistic thing to make all the lines consistent at the cost of using `id` which I expect to be optimised away by the component.  I've removed.